build_projects/shared-build-targets-utils/Utils/SharedFrameworkPublisher.cs

```diff
@@ -139,19 +139,18 @@ public void PublishSharedFramework(string outputRootDirectory, string commitHash
             PublishMutationUtilties.CleanPublishOutput(
                 sharedFrameworkNameAndVersionRoot,
                 "framework",
+                new string[] { "framework", "apphost", "hostfxr", "dotnet" },
```

Comment:
I have looked at the implementation @ramarag. Its is an invalid entry to be had in the array - why is it present as there is no such file?